data/json/monsters/feral_humans.json

```diff
@@ -45,6 +45,7 @@
       }
     ],
     "death_drops": "feral_humans_death_drops_pipe",
+    "upgrades": { "half_life": 90, "into": "mon_zombie" },
```

Comment:
Would a zombie master's forced upgrade make ferals auto-die from this? 🤔

Reply:
![image](https://user-images.githubusercontent.com/59517351/224926537-e3a84dec-093a-44ab-95df-2642d40e0eaf.png)

Wow, it looks like it behaves properly in regard with evolution. I spawned some ferals after setting the time to 5 years later and a zombie master next to them.